### ISSUE TITLE
Plugin broken with matomo 4.0.0-b2

### ISSUE THREAD

**Issue by nightvisi0n** [dominik-th/matomo-plugin-LoginOIDC#29]
Currently every sign-up ends up with:

```
An error occurred
Specify at least one website ID in &idSites=
```

If the user re-tries the error changes to:  
```
An error occurred
Username '<redacted>' already exists.
```

I think I've already identified the problem, but I have to validate that it fixes this issue.  
I will open a PR afterwards.

**Comment by dominik-th** [comment]
How did you install the plugin?
Currently only Matomo versions between 3.8.0-b4 and 4.0.0-b1 are supported.

If you know there is an issue with Matomo 4.x feel free to open a PR to the 4.x-dev branch. Keep in mind that this branch has not been released to the Matomo marketplace yet

**Comment by dominik-th** [comment]
Fixed in version 4.0.0
